### PR TITLE
Cleaning up rule c249d5 "Device motion based changes to the content can be disabled"

### PR DESCRIPTION
**Pull request by carlosapaduarte**
- Removing note that repeated what is already stated in an assumption
- Changing the time intervals to match what is in the definition of "changes in content"

Need for Final Call:
This will not require a Final Call (editorial changes)

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/device-motion-disabled-c249d5.md [hunk 1]
```diff
@@ -32,10 +32,8 @@ The rule applies to an [HTML document][] with an associated [Window object][] th
...
-
-**Note:** The 10 minutes time span is an arbitrary limit which is not included in WCAG. Results that happen after this period will not fail this rule but may nonetheless fail [Success Criterion 2.5.4: Motion Actuation][sc2.5.4]. The accessibility problem tends to be less severe for longer time periods, and without a time limit, testing this rule consistently would be impractical.
+- **no changes:** The registered event does not cause [changes to the content][changes in content] of the [web page][] within a 1 minutes time span of the [event firing][event firing]; or
+- **disabled:** There is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] of the registered event or can be found in a [clearly labeled location][] from that [web page][]. The set of [instruments][instrument] can be used to prevent [changes to the content][changes in content] of the [web page][] resulting from the event within a 1 minutes time span of the [event firing][].
```
Comment (Jym77) [inline _rules/device-motion-disabled-c249d5.md:36]: Not completely related to this PR, but this item could benefit from https://github.com/act-rules/act-rules.github.io/blob/dca29f7ac2cbde4023d389448b483872a18e40ae/pages/glossary/set-of-clearly-labelled-instruments.md and https://github.com/act-rules/act-rules.github.io/blob/dca29f7ac2cbde4023d389448b483872a18e40ae/pages/glossary/block-event.md

Using "block event" might also help to solve #1394 without getting an awful text:
"There is on [set of clearly labelled instruments][] to [block the event][] for at least YYY minutes after activating the instruments"
or something in that line.

Comment (carlosapaduarte) [reply _rules/device-motion-disabled-c249d5.md:36]: I completely agree with the proposed changes.
I guess we can wait for those definitions to be merged and then update the "device motion" rules.

Comment (Jym77) [reply _rules/device-motion-disabled-c249d5.md:36]: Sounds like a plan.

**Review by daniel-montalvo — APPROVED** [review]

File: _rules/device-motion-disabled-c249d5.md (hunk 1 above)
Comment (Jym77) [reply _rules/device-motion-disabled-c249d5.md:36]: We can use the "new" definitions for these notions.
```suggestion
- **disabled:** There is at least one [set of clearly labelled instruments][] to [block the event][blocked event][] for at least 1 minute.
```
(plus adding the refs)

**Review by Jym77 — CHANGES_REQUESTED** [review]

**Review by Jym77 — APPROVED** [review]

**Comment by carlosapaduarte** [comment]
Merging editorial changes

**Pull request merged**